client/package.json

```diff
@@ -17,7 +17,7 @@
     "web-vitals": "^0.2.4"
   },
   "scripts": {
-    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
@gregali9311 Do we want this committed to the repository or is this a change we just want to be local?